src/app/components/date-range-picker/index.html

```diff
@@ -0,0 +1,262 @@
+<sky-demo-page
+  pageTitle="Date range picker"
+  skyuxModule="SkyDateRangePickerModule"
+  packageName="@skyux/datetime"
+>
+  <sky-demo-page-summary>
+    The date range picker component creates a text input with a dropdown for users to select a date range from a set of well-known options. This includes options that enable users to specify dates to create their own date range. The text input is a composite component of up to two <stache-code>sky-datepicker</stache-code> components.
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties
+    sectionHeading="Date range picker component properties"
+  >
+    <sky-demo-page-property
+      propertyName="label"
+    >
+      Specifies a label for the date range picker. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="calculatorIds"
+      isOptional="true"
+    >
+      Specifies IDs for the date range options to include in the list that users select from. The options specify calculators to determine the dates in date ranges. This property accepts <a stacheRouterLink="." fragment="skydaterangecalculation-properties">an array of <stache-code>SkyDateRangeCalculatorId</stache-code> values</a>. By default, this property includes <a stacheRouterLink="." fragment="skydaterangecalculation-properties">all <stache-code>SkyDateRangeCalculatorId</stache-code> values</a>.
```

Comment:
I'm not seeing any anchor tags for `skydaterangecalculation-properties`. Do you mean to say `skydaterangecalculatorid-properties` ?

Reply:
I also notice this same link is repeated a couple times in this document.

Reply:
Good find.